model/clusters_mgmt/v1/root_resource.model

```diff
@@ -61,4 +61,9 @@ resource Root {
 	locator Products {
 		target Products
 	}
+
+	// Reference to the resource that manages the collection of external cluster services.
+	locator ExternalClusterServices {
```

Comment:
this can be accessed both from root and from the cluster?

Reply:
yes. The root is a general config.
And the cluster is link to the one attached to the cluster.
Kind of like flavour. Only flavour is exposed in the cluster body, and this isn't.